cmd/crank/beta/validate/cmd.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2024 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package validate implements offline schema validation of Crossplane resources.
+package validate
+
+import (
+	"github.com/alecthomas/kong"
+
+	"github.com/crossplane/crossplane-runtime/pkg/errors"
+	"github.com/crossplane/crossplane-runtime/pkg/logging"
+)
+
+// Cmd arguments and flags for render subcommand.
+type Cmd struct {
+	// Arguments.
+	Extensions string `arg:"" help:"Extensions source which can be a file, directory, or '-' for standard input."`
+	Resources  string `arg:"" help:"Resources source which can be a file, directory, or '-' for standard input."`
+
+	// Flags. Keep them in alphabetical order.
+	SkipSuccessLogs bool `help:"Skip printing success logs."`
```

Comment:
`logs` fields a bit weird here. `messages` sounds more accurate, but it is longer to type.  Maybe `results` instead?

Reply:
updated as `--skipSuccessResults`